sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/implementation/models/RandomBytes.java

```diff
@@ -41,7 +41,7 @@
      * @param bytes The random bytes to set.
      * @return The updated {@link RandomBytes} object.
      */
-    public RandomBytes setValue(byte[] bytes) {
+    public RandomBytes setBytes(byte[] bytes) {
```

Comment:
Why should a user be able to override the bytes on a purely response model? .NET's property is read-only, as this should be. Or is this one not exposed and was generated that way, which would indicate a bug in swagger.

/cc @maorleger to open an issue with KV if so.

Reply:
This is an internal model, so this won't be seen by users.